src/core/api.js

```diff
@@ -12,10 +24,25 @@ export async function fetchUsdValue() {
   const month = date.getMonth() + 1;
   const today = `${date.getDate()}-${month < 10 ? `0${month}` : month}-${date.getFullYear()}`;
   try {
-    const result = await fetch(`https://mindicador.cl/api/dolar/${today}`);
+    const result = await fetch(`${principalApi.url}${today}`);
```

Comment:
Quizás agregarle un timeout 😄 

```suggestion
    const result = await fetch(`${principalApi.url}${today}`, { timeout: 5000 });
```